pkg/k8shandler/util.go

```diff
@@ -253,7 +254,7 @@ func (er *ElasticsearchRequest) isValidConf() error {
 	}
 
 	// TODO: replace this with a validating web hook to ensure field is immutable
-	if ok, msg := hasValidUUIDs(dpl); !ok {
+	/*if ok, msg := hasValidUUIDs(dpl); !ok {
```

Comment:
this isn't going to stay in, i needed to do this to be able to recreate this issue